homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -137,6 +138,148 @@ class EnvoyProductionSensorEntityDescription(
 )
 
 
+@dataclass(frozen=True)
+class EnvoyProductionPhaseSensorEntityDescription(
+    EnvoyProductionSensorEntityDescription
+):
+    """Describes an Envoy production phase sensor entity."""
+
+    on_phase: PhaseNames = PhaseNames.PHASE_1
```

Comment:
```suggestion
@dataclass(frozen=True, kw_only=True)
class EnvoyProductionPhaseSensorEntityDescription(
    EnvoyProductionSensorEntityDescription
):
    """Describes an Envoy production phase sensor entity."""

    on_phase: PhaseNames
```

This allows you to remove the default

Reply:
Also, We now also have translation placeholders, maybe we can use that to avoid having 6 entities per phase * 3 = 18 entities. We could just have 6 in which we replace stuff. What do you think?

Reply:
Removed default and implemented translation placeholder.